Virtual Librarian/Virtual Librarian/BookCopy.cs

```diff
@@ -3,19 +3,44 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.Data.Linq;
+using System.Data.Linq.Mapping;
 
 namespace Virtual_Librarian
 {
+    [Table(Name = "BookCopies")]
     class BookCopy
     {
-        private DateTime dateOfPrinting { get; }
-        public DateTime takenDate { get; set; }
-        public DateTime? lastReturnDate { get; set; }
-        private Book book { get; }
-        public BookCopy (Book book, DateTime dateOfPrinting)
+        [Column(IsPrimaryKey = true, IsDbGenerated = true)] public int copyID { get; private set; }
+        [Column(CanBeNull = true)]  public DateTime? takenDate      { get;  set; }
+        [Column(CanBeNull = true)]  public DateTime? lastReturnDate { get;  set; }
+        [Column(CanBeNull = true)]  public DateTime? returnUntil    { get;  set; }
+        [Column(CanBeNull = false)] private string ISBN { get; set; }
```

Comment:
can you explain why are ISBN and takenByID private?

Reply:
I think